### PR TITLE
Login management

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -1,21 +1,24 @@
 import datetime
 
-from peewee import Model, CharField, DateField
-
 from core import db
```
Comment (vinyll) [inline core/models.py]: This is an internal package, it should be after the vendor packages

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+bcrypt==3.1.3
```
Comment (vinyll) [inline requirements.txt]: Should we package `bcrypt` or is it a _Playhouse_'s dependency?

File: core/models.py (hunk 1 above)
Comment (davidbgk) [reply core/models.py]: I modified the import to be consistent

File: requirements.txt (hunk 2 above)
Comment (davidbgk) [reply requirements.txt]: It has to be installed independently http://docs.peewee-orm.com/en/latest/peewee/playhouse.html#PasswordField

File: auth/views.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        flash('Logged in successfully.')
+        next = request.args.get('next')
+        if not is_safe_url(next):
+            return abort(400)
+        return redirect(next or url_for('home'))
+
+    return render_template('login.html', form=form, error=error)
+
+
+@app.route("/logout")
```
Comment (vinyll) [inline auth/views.py]: single quotes

File: readme.md [hunk 4]
```diff
@@ -31,5 +32,5 @@ re-initializing the database.
 ## Running the project
 
 ```
-python app.py
+flask run
```
Comment (vinyll) [inline readme.md]: 😞

File: settings.py [hunk 5]
```diff
@@ -1,2 +1,9 @@
 DEBUG = True
 SECRET_KEY = 'TODO'
+
+DATABASE = {
+    'name': 'cafebabel.db',
+    'engine': 'peewee.SqliteDatabase',
+}
+
+SECURITY_PASSWORD_SALT = 'and pepper'
```
Comment (vinyll) [inline settings.py]: 😆

File: templates/base.html [hunk 6]
```diff
@@ -15,10 +15,10 @@
 {% endwith %}
 {% if current_user.is_authenticated %}
     Hi {{ current_user }}!
-    <p>Do you want to <a href="{{ url_for('logout') }}">log out?</a>
+    <p>Do you want to <a href="{{ url_for('security.logout') }}">log out?</a>
```
Comment (vinyll) [inline templates/base.html]: ❔ mark out of the link

**Comment by vinyll** [comment]
r+

**Pull request merged**